lib/src/mobile_scanner_controller.dart

```diff
@@ -112,6 +113,13 @@ class MobileScannerController extends ValueNotifier<MobileScannerState> {
   /// Only supported on Android.
   final bool autoZoom;
 
+  /// If set this sets the zoom scale factor when the camera launches
```

Comment:
Suggestion for the doc comment:

```suggestion
  /// The initial zoom scale for the camera.
  ///
  /// Defaults to no initial zoom and is only supported on iOS, MacOS and Android.
```